views/files/file-viewer.hbs

```diff
@@ -1,16 +1,16 @@
 <section class="file-viewer">
-    <a href="#" class="lightbox videostop" id="file-view">
+    <a href="#" class="lightbox videostop" id="file-view" role="button" aria-label="{{$t "files.label.backToFiles"}}" tabindex="0">
       <video id="my-video" class="video-js vjs-default-skin videoclick" autoplay controls preload="auto" data-setup='{"fluid": true}'></video>
 		  <script src="/vendor-optimized/video.js/dist/video.min.js" nonce="{{nonceValue}}"></script>
 		  <script src="/vendor-optimized/video.js/dist/lang/{{ @root.userLanguage }}.js" nonce="{{nonceValue}}"></script>
-      <img id="picture">
+      <img id="picture" alt="">
     </a>
 </section>
 
 {{#embed "lib/components/modal" class="open-modal" title=($t "files.headline.doYouWantToOpenThisFile")}}
     {{#content "body"}}
         <div class="modal-body">
-            <input class="box" type="checkbox" name="allowExternalViewers" value="save">{{$t "files.text.rememberThisSelection"}}<br>
+            <input class="box" type="checkbox" name="allowExternalViewers" value="save" aria-label="{{$t "files.text.rememberThisSelection"}}">{{$t "files.text.rememberThisSelection"}}<br>
```

Comment:
actually we don't really need an aria-label here because we have a visual label which is exactly the same. It would make sense to override the visual label for screen readers if additional information was added/needed for the screen reader users.

Reply:
done